onnx/backend/test/case/node/isinf.py

```diff
@@ -40,4 +52,22 @@ def export_negative_infinity_only() -> None:
 
         x = np.array([-1.7, np.nan, np.inf, -3.6, np.NINF, np.inf], dtype=np.float32)
         y = np.isneginf(x)
-        expect(node, inputs=[x], outputs=[y], name="test_isinf_negative")
+        expect(
+            node,
+            inputs=[x],
+            outputs=[y],
+            name="test_isinf_negative",
+            opset_imports=[onnx.helper.make_opsetid("", 19)],
```

Comment:
this will prevent the test case being tested with other opset versions

Reply:
IIRC we will use the latest opset (20) by default if not specified, which means we will disable them all in ONNX CI. Any suggestions?

cc @jcwchen

Reply:
Removed

Reply:
I think we should specify opset_version here only on demand. For example, if the old opset test failed with the latest opset schema: https://github.com/onnx/onnx/blob/bc424cd4764afc7fb8a940023cdf07b6081b02e6/onnx/backend/test/case/node/loop.py#L133.